internal/cmd/tui.go

```diff
@@ -339,6 +341,41 @@ func (m tuiModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			}
 
 			return m, tea.Quit
+
+		case "ctrl+w", "w":
```

Comment:
Getting the following panic on submit:

```
Caught panic:

runtime error: index out of range [4] with length 1

Restoring terminal...

goroutine 1 [running]:
runtime/debug.Stack()
	/Users/mxs/.gvm/gos/go1.19/src/runtime/debug/stack.go:24 +0x64
runtime/debug.PrintStack()
	/Users/mxs/.gvm/gos/go1.19/src/runtime/debug/stack.go:16 +0x1c
github.com/charmbracelet/bubbletea.(*Program).Run.func1()
	/Users/mxs/.gvm/pkgsets/go1.19/global/pkg/mod/github.com/charmbracelet/bubbletea@v0.23.1/tea.go:383 +0x8c
panic({0x1059c0ce0, 0x14000426870})
	/Users/mxs/.gvm/gos/go1.19/src/runtime/panic.go:890 +0x258
github.com/stateful/runme/internal/cmd.tuiModel.View({{0x14000400000, 0x1, 0x1}, {0x1400058e9c0, 0x2c}, 0x5, 0x140007f75f0, 0x8, 0x4, {0x0, ...}})
	/Users/mxs/Projects/runme/internal/cmd/tui.go:235 +0x658
github.com/charmbracelet/bubbletea.(*Program).eventLoop(0x140007c8e70, {0x105a43550?, 0x140007e61e0?}, 0x105a3c4a0?)
	/Users/mxs/.gvm/pkgsets/go1.19/global/pkg/mod/github.com/charmbracelet/bubbletea@v0.23.1/tea.go:326 +0x5c8
github.com/charmbracelet/bubbletea.(*Program).Run(0x140007c8e70)
	/Users/mxs/.gvm/pkgsets/go1.19/global/pkg/mod/github.com/charmbracelet/bubbletea@v0.23.1/tea.go:446 +0x6b0
github.com/stateful/runme/internal/cmd.tuiCmd.func1(0x1400015a600, {0x10563e2b1?, 0x1?, 0x1?})
	/Users/mxs/Projects/runme/internal/cmd/tui.go:112 +0x714
github.com/spf13/cobra.(*Command).execute(0x1400015a600, {0x14000032050, 0x1, 0x1})
	/Users/mxs/.gvm/pkgsets/go1.19/global/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:916 +0x5c8
github.com/spf13/cobra.(*Command).ExecuteC(0x1400015a600)
	/Users/mxs/.gvm/pkgsets/go1.19/global/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x35c
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/mxs/.gvm/pkgsets/go1.19/global/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
main.root()
	/Users/mxs/Projects/runme/main.go:14 +0x128
main.main()
	/Users/mxs/Projects/runme/main.go:22 +0x1c
panic: interface conversion: tea.Model is nil, not cmd.tuiModel

goroutine 1 [running]:
github.com/stateful/runme/internal/cmd.tuiCmd.func1(0x1400015a600, {0x10563e2b1?, 0x1?, 0x1?})
	/Users/mxs/Projects/runme/internal/cmd/tui.go:117 +0xa80
github.com/spf13/cobra.(*Command).execute(0x1400015a600, {0x14000032050, 0x1, 0x1})
	/Users/mxs/.gvm/pkgsets/go1.19/global/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:916 +0x5c8
github.com/spf13/cobra.(*Command).ExecuteC(0x1400015a600)
	/Users/mxs/.gvm/pkgsets/go1.19/global/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x35c
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/mxs/.gvm/pkgsets/go1.19/global/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
main.root()
	/Users/mxs/Projects/runme/main.go:14 +0x128
main.main()
	/Users/mxs/Projects/runme/main.go:22 +0x1c
exit status 2
```

Reply:
I was able to replicate this error, but it is flaky it doesn't happen all the time :/